config/services.php

```diff
@@ -204,6 +204,7 @@
 $container->register(UserController::class, UserController::class)
     ->addArgument(new Reference(RendererInterface::class))
     ->addArgument(new Reference(UserService::class))
+    ->addArgument(new Reference(RepositoryManagerInterface::class))
```

Comment:
din cate vad, doar repo-ul de user e folosit in UserController. Daca nu ai nevoie de nici unu altu, il poti injecta direct pe ala. Te poti scapa de un nivel de dependinte.